### ISSUE TITLE
ISIC reproduce 

### ISSUE THREAD

**Issue by YasuoFly** [zhang-haojie/wesam#28]
Thank you for your excellent work! I have seen other issues mention the inability to reproduce the results in the paper, and I also used the official code to obtain a result of around 75 on the ISIC dataset, which did not reach the miou of 80 in the paper.

**Comment by zhang-haojie** [comment]
Yes, the initial experimental results are a bit inaccurate due to an error in the code. We will update the results in later versions. Thanks for your attention
